prow/plugins/label/label.go

```diff
@@ -226,19 +230,29 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	}
 
 	toRepeat := []string{}
+	isMember := false
 	if len(sigMatches) > 0 {
-		isMember, _ := gc.IsMember(ae.org, ae.login)
-		if !isMember {
-			toRepeat = ae.getRepeats(sigMatches, existingLabels)
-		}
+		isMember, _ = gc.IsMember(ae.org, ae.login)
+		toRepeat = ae.getRepeats(sigMatches, existingLabels)
 	}
 
 	if len(toRepeat) > 0 {
-		msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
-		if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
-			log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
+		if !isMember {
+			msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
+			if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
+				log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
+			}
 		}
 
+		if sc != nil {
+			//if sig matches then send a notification on slack
+			for _, sig := range sigMatches {
+				msg := fmt.Sprintf("%s\nIssue:%d, %s\nUrl:%s", ae.body, ae.issue.Number, ae.issue.Title, ae.issue.HTMLURL)
+				if err := sc.WriteMessage(plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg), "sig-"+sig[1]); err != nil {
```

Comment:
Sure, but I think that we should avoid adding unnecessary complexity like that. It would be nicer if the naming conventions were just consistent. They're close to being consistent now, there are just a couple of channels that differ from the labels.